py/jupyterlite/src/jupyterlite/addons/federated_extensions.py

```diff
@@ -57,7 +57,7 @@ def pre_build(self, manager):
         """yield a doit task to copy each federated extension into the output_dir"""
         root = ENV_EXTENSIONS
 
-        if not manager.ignore_sys_prefix:
+        if not "federated_extensions" in manager.ignore_sys_prefix:
```

Comment:
maybe each addon hoists this key, and `BaseAddon` grows a `is_sys_prefix_ignored()`?

Reply:
Yeah this sounds more flexible. And allows for future or third-party addons that might want to use components from the sys prefix.